UPGRADE-5.0.md

```diff
@@ -0,0 +1,7 @@
+# Upgrading to ibm-watson@4.0
```

Comment:
Move the `UPGRADE-4.0.md` to a wiki page and add a line in this file saying that if your version is prior to 4.0 look at that wiki

Reply:
Good idea 👍